README.md

```diff
@@ -1,2 +1,346 @@
-# lean-html5
-Learn how to use HTML5 (from scratch) to build websites/web applications!
+# Learn HTML5
+Learn how to use HTML5 (from scratch) to build websites/web applications! No previous knowledge needed.
+
+## Some history
+In 1989, Tim Berners-Lee(working at CERN then), wrote a memo proposing an Internet-based hypertext system where researchers can share documents. The browser and server software for this was created in the late 90's and later became known as the World Wide Web. The WWW was functioning using a langiage that Berners-Lee wrote - HTML.
+
+## What is HTML?
+HTML (short for **H**yper**T**ext **M**arkup **L**anguage) is basically 
+everything you see on the web everyday - every website uses HTML
+(including GitHub, Google, Facebook, your aunt's blog, etc).
+HTML is just the structure of the page - the texts, images, 
+buttons **but** it doesn't include the beautifiers for your page(CSS, BootStrap)
+although they can be inserted into the html file itself.
+
+## So HTML isn't pretty?
+Plain HTML isn't but it can be made awesome looking using styles and libraries(CSS, Bootstrap, etc) and clickable (JavaScript).
+
+## Do I need to install anything to write html?
+Theoretycally speaking - no, you don't,because you probably already have everything needed.
+You can write on anything(even Word) and save it in a .html extension.
+But truth is, if you want to make your life easier,
+you should install some aditional software - Notepad, Visual Studio Code, gedit/vim/nano, Atom, Sublime, etc.
+There are billions of text editors that will help you out but the one you need is probably Notepad++.
+> https://notepad-plus-plus.org
+
+## What resources do I need?
+Most of the knowledge you'll need, will be in this guide. But in the IT world things are changing pretty fast. So make sure you educate yourself on new innovations, best practices and technologies. Make sure to check out our resources.md file where you will find some interesting guides, books and even more additional information.
+
+## The basics 
+<ul> 
+    <li> The file you create should end with .html </li>
+    <li> To play(TO DO: find a more suitable verb) the file you just open it in a web browser (Internet Explorer, Firefox, Chrome, Safari, Opera) </li>
+    <li> HTML doesn't include styling of the page(that's what CSS is for) </li>
+</ul>
+
+Okay, so we write html in a text editor. Then how does our PC know it's a HTML file and not a poem or essay we've written? 
+Easy! HTML has it's own syntax( it's very close to the English language, don't worry) and it's own extension - **_.html_**.
+
+## HTML head and body
+
+Every html starts with the tag 
+``` html
+<!DOCTYPE html>
```

Comment:
👍 Did a little check on the accessibility to see if there was anything around doctype and this is perfect. 
https://www.paciellogroup.com/blog/2012/04/html-data-for-the-masses-doctype-usage/